### DISCUSSION TITLE
07167CDD-7E3B-17CE-12DA-3718A6FC242B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#97] [General]
# Unconfirmed Exoplanets

Catalogue of unconfirmed exoplanets and exomoons, supplementary to Celestia's default catalogue of confirmed exoplanets

https://celestia.mobi/addon?item=07167CDD-7E3B-17CE-12DA-3718A6FC242B

**Comment by chirochrone** [comment]
update the addon already, it's been over 1 year lmao :trollface:

**Comment by chirochrone** [comment]
problem? :trollface:

**Reply by TheLostProbe** [reply]
how tf did bro get a trollface emoji in here 💀💀

**Comment by AU-Microscoppi-b** [comment]
Question:how many exoplanets are in this addon? What are they?

**Reply by TTarrants** [reply]
Hey there! The most recent version of the addon (which you can find here: https://celestiaproject.space/forum/viewtopic.php?p=163529#p163529) contains 9776 unconfirmed planets. I won't list them for obvious reasons, but the overwhelming majority of them are transiting planet candidates from TESS, Kepler, OGLE-TR, CoRoT, and other surveys that are awaiting confirmation. There's also various RV-detected candidates.
